testing/dosbox/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Maintainer: Jakub Skrzypnik <j.skrzypnik@openmailbox.org>
+pkgname=dosbox
+_cvstag=4003
+pkgver=r${_cvstag}
+pkgrel=0
+pkgdesc="x86/DOS emulator with sound and graphics"
+url="http://dosbox.com"
+arch="all"
+license="GPLv2+"
+depends="hicolor-icon-theme"
+depends_dev="libpng-dev sdl-dev alsa-lib-dev glu-dev"
+makedepends="$depends_dev paxctl autoconf automake"
+install=""
+subpackages="$pkgname-doc"
+source="https://sourceforge.net/code-snapshots/svn/d/do/dosbox/code-0/dosbox-code-0-4003.zip 
+	dosbox.desktop
+	dosbox.png
+	"
+
+_builddir="$srcdir"/dosbox-code-0-${_cvstag}/dosbox/trunk/ 
+prepare() {
+	local i
+	cd "$_builddir"
+	update_config_sub
+}
+
+build() {
+	cd "$_builddir"
+	./autogen.sh
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		--localstatedir=/var \
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$_builddir"
+	make DESTDIR="$pkgdir" install || return 1
+	paxctl -c -M "$pkgdir"/usr/bin/dosbox
```

Comment:
Use paxmark here. It does not take the '-c' argument. But should be otherwise be flag compatible. paxmark adds xattr markings which are now the standard.